app/extensions/brave/locales/en-US/adblock.properties

```diff
@@ -1,9 +1,10 @@
-adblockTitle=Ad Block
 adblock=Ad Block
-lastUpdateCheckDateLabel=Last update check:
-lastCheckETagLabel=Last check ETag:
-blockedCountLabel=Blocked count:
+adblockTitle=Ad Block
+adblockTooManyListsWarning=Warning: Turning on too many lists will degrade performance
 additionalFilterLists=Additional filter lists
-customFilters=Custom filters
+blockedCountLabel=Blocked count:
 customFilterDescription=One per line, a filter is described in Adblock Plus filter syntax
-adblockTooManyListsWarning=Warning: Turning on too many lists will degrade performance
+customFilters=Custom filters
+disableAdBlockForSite=Disable Ad block
+lastCheckETagLabel=Last check ETag:
+lastUpdateCheckDateLabel=Last update check:
```

Comment:
@srirambv I mean like this:
````
label_lastUpdateCheckDate=...
label_lastCheckETag=...
````

Specifying a hierarchy with dashes or something more proper would make it easier for us to see the system of the localization strings and enhance it when new ones get added.

Reply:
@luixxiul I'd suggest creating a new issue and proposing the strategy. If it's something that is easier for translators and also for programmers, then maybe we can adapt it. Personally, I don't know how helpful it would be